.devcontainer/devcontainer.json

```
@@ -1,19 +1,66 @@
 {
-    "dockerComposeFile": ["../development/docker-compose.yml", "../development/docker-compose.debug.yml"],
+    "name": "Nautobot Dev Container",
+    "dockerComposeFile": [
+        "../development/docker-compose.yml",
+        "../development/docker-compose.postgres.yml",
+        "../development/docker-compose.dev.yml",
+        "../development/docker-compose.override.yml"
+    ],
+    "features": {
+        "ghcr.io/devcontainers/features/docker-outside-of-docker:1": { }
+    },
     "service": "nautobot",
-	"workspaceFolder": "/source/",
-	"extensions": [
-		"ms-python.vscode-pylance",
-		"streetsidesoftware.code-spell-checker",
-		"eamodio.gitlens",
-		"oderwat.indent-rainbow",
-		"ms-python.python",
-		"ms-vsliveshare.vsliveshare"
-	],
-	"settings": {
-		"python.pythonPath": "/usr/local/bin/python",
-		"python.analysis.extraPaths": ["/source"],
-		"python.linting.pylintEnabled": true,
-		"python.linting.enabled": true
-	}
+    "shutdownAction": "stopCompose",
+    "customizations": {
+        "vscode": {
+            "extensions": [
+                "charliermarsh.ruff",
+                "DavidAnson.vscode-markdownlint",
+                "dbaeumer.vscode-eslint",
+                "eamodio.gitlens",
+                "EditorConfig.EditorConfig",
+                "esbenp.prettier-vscode",
```

```suggestion
                "eamodio.gitlens",
                "EditorConfig.EditorConfig",
```
Since we removed the js frontend, prettier is complaining about not finding a package.json and eslint isn't needed anymore.